src/css/popup.css

```diff
@@ -470,13 +470,34 @@ manage things like container crud */
   inline-size: var(--inline-button-size);
   justify-content: center;
   text-decoration: none;
+  width: 200px;
```

Comment:
We should use `inline-size` instead of `width`

Reply:
Hi @groovecoder, when you type "Personal" it will focus on the Personal container in the popup itself as you can see in the following snapshot:
![Screenshot from 2019-10-30 10-16-05](https://user-images.githubusercontent.com/31931348/67829795-78fd1580-fafe-11e9-9d24-d3d7e0dc7093.png)

Though I've not added the same for focussing on the sites assigned to a particular container. This I am planning to do in a separate PR that I will open in a short while. I think this functionality is something that is really useful when a user has created more than 8-9 containers so this can be merged for now.